### PR TITLE
Split controllers and api definitions into multiple files

### PR DESCRIPTION
**Pull request by fguillot**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ script:
   - tox -r
   - docker-compose -f docker-compose-integration-tests.yml build
   - docker-compose -f docker-compose-integration-tests.yml up -d api collector messaging database
+  - sleep 2
```
Comment (Marx314) [inline .travis.yml]: really?

File: tests/controller/test_volume_controller.py [hunk 2]
```diff
@@ -434,15 +160,15 @@ def test_create_volume_raises_bad_date_format(self):
...
-            DateFormatException,
-            self.controller.create_volume,
-            some_volume.entity_id,
-            some_volume.project_id,
-            'bad_date_format',
-            some_volume.volume_type,
-            some_volume.size,
-            some_volume.name,
-            some_volume.attached_to
+                DateFormatException,
```
Comment (maxc0c0s) [inline tests/controller/test_volume_controller.py]: Thats a lot of spaces :)

File: tests/controller/test_volume_controller.py [hunk 3]
```diff
@@ -555,9 +281,9 @@ def test_volume_attach_with_no_existing_attachment(self):
          .with_args(fake_volume))
 
         self.controller.attach_volume(
-            fake_volume.entity_id,
-            date.strftime("%Y-%m-%dT%H:%M:%S.%f"),
-            ["new_attached_to"]
+                fake_volume.entity_id,
+                date.strftime("%Y-%m-%dT%H:%M:%S.%f"),
+                ["new_attached_to"]
```
Comment (mbelang) [inline tests/controller/test_volume_controller.py]: here as well too much spaces

File: tests/controller/test_volume_controller.py [hunk 4]
```diff
@@ -579,9 +305,9 @@ def test_volume_attach_with_existing_attachments(self):
          .with_args(fake_volume))
 
         self.controller.attach_volume(
-            fake_volume.entity_id,
-            date.strftime("%Y-%m-%dT%H:%M:%S.%f"),
-            ["existing_attached_to", "new_attached_to"]
+                fake_volume.entity_id,
+                date.strftime("%Y-%m-%dT%H:%M:%S.%f"),
+                ["existing_attached_to", "new_attached_to"]
```
Comment (mbelang) [inline tests/controller/test_volume_controller.py]: here as well too much spaces

File: tests/controller/test_volume_controller.py [hunk 5]
```diff
@@ -617,9 +343,9 @@ def test_volume_attach_after_threshold(self):
          .once())
 
         self.controller.attach_volume(
-            fake_volume.entity_id,
-            date.strftime("%Y-%m-%dT%H:%M:%S.%f"),
-            ["new_attached_to"]
+                fake_volume.entity_id,
+                date.strftime("%Y-%m-%dT%H:%M:%S.%f"),
+                ["new_attached_to"]
```
Comment (mbelang) [inline tests/controller/test_volume_controller.py]: here as well too much spaces

**Comment by mbelang** [comment]
:+1:

**Comment by Marx314** [comment]
👍

**Pull request merged**